sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -54,14 +69,40 @@ public partial class KeyVaultBackupClient
         public virtual Azure.Security.KeyVault.Administration.RestoreOperation StartSelectiveRestore(string keyName, System.Uri backupFolderUri, string sasToken, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Security.KeyVault.Administration.RestoreOperation> StartSelectiveRestoreAsync(string keyName, System.Uri backupFolderUri, string sasToken, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class KeyVaultBackupClientOptions : Azure.Core.ClientOptions
+    public partial class KeyVaultPermission
     {
-        public KeyVaultBackupClientOptions(Azure.Security.KeyVault.Administration.KeyVaultBackupClientOptions.ServiceVersion version = Azure.Security.KeyVault.Administration.KeyVaultBackupClientOptions.ServiceVersion.V7_2_Preview) { }
-        public Azure.Security.KeyVault.Administration.KeyVaultBackupClientOptions.ServiceVersion Version { get { throw null; } }
-        public enum ServiceVersion
-        {
-            V7_2_Preview = 1,
-        }
+        public KeyVaultPermission() { }
+        public System.Collections.Generic.IList<string> Actions { get { throw null; } }
```

Comment:
So we decided not to use Allow\* on these other two?

Reply:
I think Allow makes sense. I'll add that change as well.